src/commands/convertConnections/ConvertSettingsStep.ts

```diff
@@ -0,0 +1,210 @@
+            }
+
+            if (context.local) {
+                const localSettings = await AzExtFsExtra.readJSON<ILocalSettingsJson>(nonNullProp(context, 'localSettingsPath'));
+                if (localSettings.Values) {
+                    for (const connection of context.convertedConnections) {
+                        localSettings.Values[connection.name] = connection.value;
+                        if (localSettings.Values[nonNullProp(connection, 'originalValue')]) {
+                            delete localSettings.Values[nonNullProp(connection, 'originalValue')];
+                            context.activityChildren?.push(
+                                new GenericTreeItem(undefined, {
+                                    contextValue: createUniversallyUniqueContextValue(['useExistingResourceGroupInfoItem', activitySuccessContext]),
+                                    label: localize('deletedSetting', `Deleted local setting "${connection.originalValue}"`),
+                                    iconPath: activitySuccessIcon
+                                })
+                            )
+                        }
+                        context.activityChildren?.push(
+                            new GenericTreeItem(undefined, {
+                                contextValue: createUniversallyUniqueContextValue(['useExistingResourceGroupInfoItem', activitySuccessContext]),
+                                label: localize('addedSetting', `Added Local setting "${connection.name}"`),
+                                iconPath: activitySuccessIcon
+                            })
+                        );
+                    }
+                    await AzExtFsExtra.writeJSON(nonNullProp(context, 'localSettingsPath'), localSettings);
+                }
+            } else {
+                if (context.functionapp) {
+                    const client = await context.functionapp?.site.createClient(context);
+                    const remoteSettings = await client.listApplicationSettings();
+                    for (const connection of context.convertedConnections) {
+                        if (remoteSettings.properties) {
+                            remoteSettings.properties[connection.name] = connection.value;
+                            delete remoteSettings.properties[nonNullProp(connection, 'originalValue')];
+                        } else {
+                            await client.updateApplicationSettings({ properties: { [connection.name]: connection.value } });
+                        }
+                        context.activityChildren?.push(
+                            new GenericTreeItem(undefined, {
+                                contextValue: createUniversallyUniqueContextValue(['useExistingResourceGroupInfoItem', activitySuccessContext]),
+                                label: localize('deletedSetting', `Deleted app setting "${connection.originalValue}"`),
```

Comment:
Should all the activity child labels be present tense instead of past tense?

Left a similar comment to Nathan as well:
https://github.com/microsoft/vscode-azurefunctions/pull/4382#discussion_r1978361356